packages/firebase_auth/README.md

```diff
@@ -83,11 +83,35 @@ _handleSignIn()
     .catchError((e) => print(e));
 ```
 
+### Register a user
+
+```dart
+final FirebaseUser user = await _auth.createUserWithEmailAndPassword(
+      email: 'an email',
+      password: 'a password',
+    );
+```
+
+### Supported Firebase authentication methods
```

Comment:
For this list I think you don't need backticks, and I'd remove "authentication" since it applies to everything on the list